nilearn/maskers/multi_nifti_masker.py

```diff
@@ -356,12 +356,14 @@ def transform_imgs(
             target_fov=target_fov,
             memory=self.memory,
             memory_level=self.memory_level,
-            verbose=self.verbose,
         )
 
         if confounds is None:
             confounds = itertools.repeat(None, len(imgs_list))
 
+        if sample_mask is None:
+            sample_mask = itertools.repeat(None, len(imgs_list))
+
```

Comment:
here  I followed the same implementation as for maskers that actually make use of this sample_mask

Reply:
- will open an issue to add a test for this as part of #4388